aws/resource_aws_backup_plan_test.go

```diff
@@ -625,3 +760,109 @@ resource "aws_backup_plan" "test" {
 }
 `, rName)
 }
+
+func testAccAwsBackupPlanConfig_copyAction(rName1 string, rName2 string) string {
+	return fmt.Sprintf(`
+resource "aws_backup_vault" "test" {
+  name = %[1]q
+}
+
+resource "aws_backup_vault" "test2" {
+  name = %[2]q
+}
+
+resource "aws_backup_plan" "test" {
+  name = %[1]q
+
+  rule {
+    rule_name         = %[1]q
+    target_vault_name = "${aws_backup_vault.test.name}"
+    schedule          = "cron(0 12 * * ? *)"
+
+    lifecycle {
+      cold_storage_after = 30
+      delete_after       = 180
+    }
+
+    copy_action {
+      lifecycle {
+        cold_storage_after = 30
+        delete_after       = 180
+	  }
+      destination_vault_arn = "${aws_backup_vault.test2.arn}"
+    }
+  }
+}
+`, rName1, rName2)
+}
+
+func testAccAwsBackupPlanConfig_copyActionUpdated(rName1 string, rName2 string) string {
```

Comment:
Nit: Can remove this mostly duplicated configuration by parameterizing `cold_storage_after` and `delete_after`, e.g.

```
lifecycle {
  cold_storage_after = %[3]d
  delete_after       = %[4]d
}
```